tests/test_phonons_cli.py

```diff
@@ -0,0 +1,117 @@
+"""Test phonons commandline interface."""
+
+from pathlib import Path
+
+from typer.testing import CliRunner
+
+from janus_core.cli.janus import app
+
+DATA_PATH = Path(__file__).parent / "data"
+
+runner = CliRunner()
+
+
+def test_help():
+    """Test calling `janus phonons --help`."""
+    result = runner.invoke(app, ["phonons", "--help"])
+    assert result.exit_code == 0
+    assert "Usage: janus phonons [OPTIONS]" in result.stdout
+
+
+def test_phonons(tmp_path):
```

Comment:
Where is `tmp_path` coming from? Is this part of `typer`? If so, pretty neat.

Reply:
No, it's built in to pytest: https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

It is very useful